packages/azure-kusto-ingest/src/fileDescriptor.browser.ts

```diff
@@ -50,4 +56,15 @@ export class FileDescriptor extends AbstractDescriptor implements FileDescriptor
     getCompressionSuffix() {
         return this.compressionType ? `.${this.compressionType}` : ".gz";
     }
+
+    private _calculateSize(modifier: number = 1): void {
+        if (this.size == null || this.size <= 0) {
```

Comment:
actually i dont think this works - i dont think size here is evet null .. as we take the file size as the size by default